docker/ssl/Makefile

```diff
@@ -1,23 +1,25 @@
 CRT_LOCATION = certs
-THING_KEY = d7cc2964-a48b-4a6e-871a-08da28e7883d # example value
 O = Mainflux
-OU = mainflux
+OU_CA = mainflux_ca
+OU_CRT = mainflux_crt
```

Comment:
maybe there is no need to add additional variables, since `make ca` and `make server_cert` are consecutive commands we can just update docs and specify that commands should be ran with different values

Reply:
Yes, but that adds a further step to get the thigns going. Plus, if you just try to run the script, you'll be screwed and you won't know why.